### ISSUE TITLE
Being "professional"

### ISSUE THREAD

**Issue by chrisgraham** [ContributorCovenant/contributor_covenant#791]
> Other conduct which could reasonably be considered inappropriate in a professional setting

I think it's problematic to talk about making Open Source "professional" without further qualification.

Often people in Open Source are contributing informally as a hobby, and don't want to have to meet the same bar that they'd have to meet in the workplace. Of course there are inappropriate behaviors we need to present, but the set of things that are "unprofessional" is much wider than that.

For example:
- Wearing T-shirts from edgy rock bands
- Mild non-sexualized swearing
- Harmless religious quotes in signatures
- A lot of things in general that convey personal cultural preferences strongly

Many workplaces would come down very hard on all these things, but I think for most Open Source projects they would be acceptable.

I'd consider changing the sentence substantially to something like:
> Other behavior that would not be tolerated in reasonable professional settings due to the likelihood of creating significant personal discord

**Comment by avats-dev** [comment]
Hey @chrisgraham , I want to do this. 
> I think I have to edit this [here](https://github.com/ContributorCovenant/contributor_covenant/blob/release/CODE_OF_CONDUCT.md#our-standards), right?

**Comment by chrisgraham** [comment]
Yes, but I don't know if the project is going to accept changes to the meaning of the document at the moment given the large number of translations and projects adopting it. So I'd check that first if I were you.

**Comment by Chealer** [comment]
This is more or less a duplicate of ticket #39, but the rationale is different. What you seem to be arguing is that most of the manpower of projects which would consider this covenant are from volunteers, which means they should be allowed certain behaviors which may not be acceptable in most remunerated jobs, such as reasonable advertising/evangelizing.

I do not disagree with your assessment that the audience is statistically different, and it may be logical to lower standards indeed. That being said, it is unclear where to set the bar. I am not convinced by your suggested wording. The likelihood of creating discord seems like a poor measure of acceptability; speaking one's mind suffices to likely create discord. I would rather avoid using professionalism―whatever it means―as a reference.
